changelogs/unreleased/build-master-fix-flaky-tests-relying-on-ordered-data.yml

```diff
@@ -0,0 +1,3 @@
+description: Fix flaky tests relying on ordered data
+change-type: minor
+destination-branches: [master, iso6, iso5]
```

Comment:
Discussion on slack suggests this get fixed on all required product lines, are there any I'm missing here ?

Reply:
REsource sets were introduced in iso5. So I think this is enough.